### PR TITLE
CNV-25427: Release note NEW - UI - Enable SSH over LB

### PR DESCRIPTION
**Pull request by danielclowers**
**Version(s):**
4.13

**Issue:**
https://issues.redhat.com/browse/CNV-25427

**Link to docs preview:**
https://57805--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-13-release-notes.html#virt-4-13-web-new

**QE review:**
- [x] QE has approved this change.

**Additional information:**
NA

### CODE REVIEW

File: virt/virt-4-13-release-notes.adoc [hunk 1]
```diff
@@ -88,6 +88,7 @@ The SVVP Certification applies to:
 //CNV-22555 Release note: CHANGE Hostname field moving from cloud-init options to VM details page
 
 //CNV-25427 Release note: NEW Allow VM admin to expose SSH service using LB in the UI
+* The *VirtualMachine details* -> *Details* tab now provides an option to select *SSH over LoadBalancer* when exposing a Kubernetes services.
```
Comment (gouyang) [inline virt/virt-4-13-release-notes.adoc:91]: What do you mean `exposing a Kubernetes services` here? It confuses me a bit.

Comment (danielclowers) [reply virt/virt-4-13-release-notes.adoc:91]: Perhaps we can remove `a Kubernetes`. [This was the article](https://docs.openshift.com/container-platform/4.12/virt/virtual_machines/vm_networking/virt-creating-service-vm.html#virt-about-services_virt-creating-service-vm) I was referring to when I chose the terminology that I did, however if it's confusing we can simply say _services_, like below:

"The *VirtualMachine details* -> *Details* tab now provides an option to select *SSH over LoadBalancer* when exposing services."

Comment (gouyang) [reply virt/virt-4-13-release-notes.adoc:91]: What about?
The VirtualMachine details -> Details tab now provides a new SSH service type "SSH over LoadBalancer" to expose the SSH service over LoadBalancer.

Comment (danielclowers) [reply virt/virt-4-13-release-notes.adoc]: I changed it to what was suggested. Does that seem fine now?

File: virt/virt-4-13-release-notes.adoc [hunk 2]
```diff
@@ -88,6 +88,7 @@ The SVVP Certification applies to:
 //CNV-22555 Release note: CHANGE Hostname field moving from cloud-init options to VM details page
 
 //CNV-25427 Release note: NEW Allow VM admin to expose SSH service using LB in the UI
+* The *VirtualMachine details* -> *Details* tab now provides a new SSH service type *SSH over LoadBalancer* to expose the SSH service over LoadBalancer.
```
[the remark this answers is not in the document]
Comment (danielclowers) [reply virt/virt-4-13-release-notes.adoc]: Thanks, I agree with the suggested formatting. Thanks also for that API object formatting link!

**Review by maxwelldb — APPROVED** [review]

**Comment by maxwelldb** [comment]
@danielclowers Did you need to get a QE ack for this? I note that this is unresolved: https://github.com/openshift/openshift-docs/pull/57805#discussion_r1150629690

File: virt/virt-4-13-release-notes.adoc (hunk 1 above)
Comment (danielclowers) [reply virt/virt-4-13-release-notes.adoc]: @gouyang do the changes look good at this point?

**Pull request merged**